src/utils/outputs/swiftui/shared.ts

```diff
@@ -0,0 +1,238 @@
+export const designSystemName = "DesignSystem"
+
+export const densities = ["Functional", "Regular", "Expressive"];
+
+export const devices = ["Mobile", "Tablet"];
+
+export const shirtSizes = [
+  "3xs",
+  "2xs",
+  "xs",
+  "sm",
+  "md",
+  "lg",
+  "xl",
+  "2xl",
+  "3xl",
+];
+
+export const generateStaticSwiftUIFiles = (): string => {
```

Comment:
Those are all `color` related, why did you add it to the `shared.ts` file instead of `colors.ts`?